docs/languages/powershell.md

```diff
@@ -0,0 +1,128 @@
+# Editing PowerShell in Visual Studio Code
+
+PowerShell is a task-based command line shell and scripting language built on the .NET framework.
+Powershell provides a powerfull toolset for administrators on any platform.
+
+The PowerShell extension provides rich [PowerShell Language Support](https://github.com/PowerShell/vscode-powershell)
+
+## Install the PowerShell extension on Visual Studio Code
+
+### Installing the Extension
+
+The official extension can be installed by following the steps in the
+[Visual Studio Code Docs](https://code.visualstudio.com/docs/editor/extension-gallery)
+or by going directly to the Marketplace and clicking the [Install Button](vscode:extension/ms-vscode.PowerShell)
+
+You can also install the Extension from within Visual Studio code by
+browsing
+
+![Extension Browser](images/powershell/extensionBrowser.png)
+
+ for extensions `(ctl+Shift+x)`  and searching for PowerShell. Select the PowerShell extension for details
```

Comment:
Typo: ctrl

Reply:
Also extra whitespace before `and` and should have linebreak after `PowerShell.`